csharp/extractor/Semmle.Extraction.CSharp/Entities/Types/Type.cs

```diff
@@ -272,20 +272,21 @@ public void ExtractRecursive(TextWriter trapFile, IEntity parent)
             ExtractRecursive();
         }
 
-        public static Type Create(Context cx, ITypeSymbol type)
+        // todo vt: return type is nullable
+        public static Type Create(Context cx, ITypeSymbol? type)
         {
             type = type.DisambiguateType();
-            const bool errorTypeIsNull = false;
-            return type == null || (errorTypeIsNull && type.TypeKind == TypeKind.Error) ?
-                NullType.Create(cx).Type : (Type)cx.CreateEntity(type);
+            return type is null
```

Comment:
This seems a bit strange. If `type is null` we return a non-null entry. But if it's not `null`, then the returned value might be `null`.